### ISSUE TITLE
Missing code for pair-pm

### ISSUE THREAD

**Issue by bd317** [RLHFlow/RLHF-Reward-Modeling#50]
For pair_pm train.py is missing. It is mentioned in the readme and I also checked previous commits but I can´t find that file.

Thank in advance.

**Comment by WeiXiongUST** [comment]
Hi, the training is a standard supervised finetuning so we can use the axolotl package. To do this, we only need a yaml file to provide the dataset, model, and parameter. Then, you can just use the command like CUDA_VISIBLE_DEVICES="0,1,2,3" torchrun --nproc_per_node 4 --master_port 20001 -m axolotl.cli.train gemma-2b-it.yaml to do the training
